package.json

```diff
@@ -4,7 +4,9 @@
   "description": "openaps oref0 reference implementation of the reference design",
   "scripts": {
     "test": "make test",
-    "global-install": "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0"
+    "deprecated-global-install" : "npm install && sudo npm install -g && sudo npm link && sudo npm link oref0",
+    "global-install": "Error: global-install is deprecated. Please use 'npm install -g oref0' for a packaged version, or 'npm install -g .' for a git checkout. See https://github.com/openaps/oref0/pull/302",
```

Comment:
Sounds like we're trying to accomplish the same thing: we want something like `npm run global-install` (off a manual clone) for developers who need it, but for regular users we want everything to be installed via packages, with some other method (curl/wget would probably work, although not sure it's the cleanest) to pull in files we need that npm can't install.  The trick, I think, is making sure that if a developer does `npm run global-install`, that we don't overwrite that with a packaged install...